style.css

```diff
@@ -0,0 +1,72 @@
+html{
+    background-color: #ffdd99;
+}
+
+.header{
+    width: 100%;
+    color: #ffdd99;
+    background-color: #336600;
+    text-align: center;
```

Comment:
Text-align: center is used in multiple places. You can create a new class just for this. Something like this:

.align-center {
  text-align: center;
}

and use this class on every element that you want to be centered

Reply:
I think it makes the html code look much cleaner if i just put all i need in one class and then use it for one element:
<header class="header">      instead of       <header class="header" class="align-center">

Reply:
For now, it might seem it looks cleaner. But in the future, when you'll add html from javascript, you'll have less javascript code. For example, the list of contacts will only be a for loop in javascript with 1-2 lanes of html code